src/QMCHamiltonians/QMCHamiltonian.h

```diff
@@ -43,6 +43,7 @@ class QMCHamiltonian
   typedef QMCHamiltonianBase::RealType RealType;
   typedef QMCHamiltonianBase::ValueType ValueType;
   typedef QMCHamiltonianBase::Return_t Return_t;
+  typedef QMCHamiltonianBase::Return_ct Return_ct;
```

Comment:
Is Return_ct type still used? If not, remove all

Reply:
removed.